parseAPI/src/IA_amdgpu.h

```diff
@@ -49,22 +49,23 @@ class IA_amdgpu : public IA_IAPI {
                Dyninst::InstructionSource *isrc,
 	       Dyninst::ParseAPI::Block * curBlk_);
 	IA_amdgpu(const IA_amdgpu &);
-	virtual IA_amdgpu* clone() const;
-    virtual bool isFrameSetupInsn(Dyninst::InstructionAPI::Instruction) const;
-	virtual bool isNop() const;
-	virtual bool isThunk() const;
-	virtual bool isTailCall(const ParseAPI::Function* context, ParseAPI::EdgeTypeEnum type,
-							unsigned int, const set<Address>& knownTargets) const;
-	virtual bool savesFP() const;
-	virtual bool isStackFramePreamble() const;
-	virtual bool cleansStack() const;
-	virtual bool sliceReturn(ParseAPI::Block* bit, Address ret_addr, ParseAPI::Function * func) const;
-	virtual bool isReturnAddrSave(Address& retAddr) const;
-	virtual bool isReturn(Dyninst::ParseAPI::Function * context, Dyninst::ParseAPI::Block* currBlk) const;
-	virtual bool isFakeCall() const;
-	virtual bool isIATcall(std::string &) const;
-	virtual bool isLinkerStub() const;
-	virtual bool isNopJump() const;
+	IA_amdgpu* clone() const override;
```

Comment:
Let's not fix the `-Winconsistent-override` warnings in this PR. Instead, just don't explicitly mark `isAbort` as `override`. All of the `InstructionAdaptor` classes need to be cleaned up like this, though.